src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -89,6 +106,20 @@ function AnchorRenderer({tnode, key, style}) {
         );
     }
 
+    if (!isInsideComment(tnode)) {
+        // This is not a comment from a chat
```

Comment:
Can you update this comment to also explain why we want to return something different if the node is not a chat comment?

Reply:
Thanks Joe, I'll check that 😉